src/coord/timestamp.rs

```diff
@@ -44,6 +45,9 @@ use crate::coord;
 use futures::executor::block_on;
 use itertools::Itertools;
 
+/// Number of seconds that must elapse before we update watermarks again
+const WATERMARK_UPDATE_INTERVAL: u64 = 1;
```

Comment:
More importantly though: Can you name this something that explicitly includes "metrics". Otherwise a casual reader might think this is controlling the Rt timestamper watermark update rate, which is still 10ms by default (and actually needed for the timestamping logic)

Reply:
ah fair.